exampleSite/content/blog/_index/list.md

```diff
@@ -5,5 +5,5 @@ weight = 100
 
 #count = 10 # Default value is 10
 #section = "" # Default value is current section
-#disable_summary = true
+#summary = true
```

Comment:
The default is already true so the comment should probably be false. Also adding a small comment that summary defaults to true, when not set would be good for the user.